src/bgw/job_stat.c

```diff
@@ -191,25 +201,36 @@ calculate_next_start_on_failure(TimestampTz last_finish, int consecutive_failure
 	float8 jitter = calculate_jitter_percent();
 	/* consecutive failures includes this failure */
 	float8 multiplier = 1 << (consecutive_failures - 1);
+	Timestamp res;
```

Comment:
```suggestion
	volatile Timestamp res;
```
if we're going to change it in both branches of the `try/catch` this is required.

Reply:
Also should this be a `TimestapTz`? That's what it used to be...

Reply:
yes should be TimestampTz